RELEASE_NOTES.md

```diff
@@ -1,94 +1,98 @@
 # Release Notes
 
+NOTE: This is the draft for the releases notes. If you are an implementer or someone that is upgrading a Decidim installation, you need to follow
+the instructions for all the patch releases in GitHub:
+
+- https://github.com/decidim/decidim/releases/tag/v0.31.0.rc1
+
 ## 1. Upgrade notes
 
 As usual, we recommend that you have a full backup, of the database, application code and static files.
 
 To update, follow these steps:
 
-### 1.1. Update your ruby version
+### 1.1. Update your ruby and node versions
 
 If you're using rbenv, this is done with the following commands:
 
 ```console
-rbenv install 3.x.x
-rbenv local 3.x.x
+rbenv install 3.3.4
+rbenv local 3.3.4
 ```
 
 You may need to change your `.ruby-version` file too.
 
-If not, you need to adapt it to your environment, for instance by changing the decidim docker image to use ruby:3.x.x.
-
-### 1.2. Update your application configuration
```

Comment:
These instructions were too noisy. I prefer to have a TLDR version with "links" (aka mentions) to the section that better explains the change (see my proposal on "### 1.3. Run these commands")